native/src/modules/endpoint/sagas/__tests__/watchFetchEvent.spec.js

```diff
@@ -47,7 +91,7 @@ describe('watchFetchEvents', () => {
           key: 'events-key',
           criterion: {
             forceUpdate: false,
-            shouldRefreshResources: true
+            shouldRefreshResources: false
```

Comment:
Why did this change?

Reply:
We now have one test which refreshes content and one that doesn't. This is the one that doesn't.